assets/shared.css

```diff
@@ -35,6 +38,53 @@ a {
     color: var(--link-color);
     text-decoration: none;
     font-weight: bold;
+    transition: 250ms ease-in;
+}
+
+a:hover {
+    color: var(--link-active-color);
+    text-decoration: underline;
+    transition: 250ms ease-out;
+}
+
+.generic-button {
+    padding: 0.8em;
+    margin-top: 1em;
+
+    font-weight: bold;
+    font-size: 18px;
+
+    background: var(--input-color);
+    color: var(--text-color);
+
+    border: hidden;
+    border-radius: 10px;
+
+    transition: 250ms ease-in;
+}
+
+.generic-button:hover {
+    background-color: #8d409c;
+    transition: 250ms ease-out;
+}
+
+.thin-button.thin-button {
+    padding: 0.5em;
+    margin-top: 1em;
+}
+
+.thick-button.thick-button {
+    padding: 1em;
+    margin-top: auto;
+}
+
+.results-table {
+    border-collapse: collapse;
+}
+
+.results-header th {
```

Comment:
Padding is fine but let's leave the color out for now. Also, the same class must be applied to all the `td`, or it doesn't align, so it probably needs a different name like `results-column`.